sartography/spiff-arena
# Feature/pi short link

This adds a link icon to the process instance show page that can copy a short link to the user's clipboard. The link will be like: `http://localhost/i/27`. This link will be the row of icons that also has "suspend" and "delete". The feature uses the find-by-link api in the backend so it will automatically determine if the show page should be for the "for-me" page or not.

This PR will also remove the "/admin" prefix from all pages that had it. So instead of a path being like "/admin/process-groups" it will now be "/process-groups".

To Test:
* go to the instance show page and click on the copy link icon and paste in a new tab. it should navigate you to the same process instance.
